.github/workflows/codeql-analysis.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
       - name: Build artifacts
-        run: ./gradlew build
+        run: ./gradlew build -x test
```

Comment:
Det var lurt! 💡 For du sparer vel litt tid her? I så fall skal jeg endre dette i de andre appene også :D

Reply:
Det går litt raskere ja 🙂  jeg har gjort det i noen, litt etterhvert som jeg blir irritert på den 😅